diff --git a/node_modules/multinodejs/lib/commands.js b/node_modules/multinodejs/lib/commands.js
index a368d5c..0c0e3b6 100644
--- a/node_modules/multinodejs/lib/commands.js
+++ b/node_modules/multinodejs/lib/commands.js
@@ -82,6 +82,11 @@ module.exports = {
     // use the fourth parameters of each field for Upgrade
     create: ["stream|streamfilter|txfilter|upgrade", "name", "open|restrictions|restrictions|false", {"custom-fields|js-code|js-code|params": {}}],
 
+    getvariablevalue: ["variable"],
+    getvariablehistory: ["variable"],
+    setvariablevalue: ["variable", "value"],
+    listvariables: ["variables", "verbose", "count", "start"],
+
     // createFrom is also used on Filters & Upgrade,
     // use the second parameters for that
     createFrom: ["from-address", "type=stream|type=filter/upgrade", "name", "open/params|open=false", {"custom-fields|protocol-version": {}}],
